content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,6 +7,20 @@ menu:
     name: Release notes
 ---
 
+## v1.6.3 [2022-01-20]
+
+### Features
+
+- Support for the `attributes` field in Alerta node.
+- Add BigPanda handler options.
+- The `env var` config can now handle new items.
+- `Topic queue length` is now configurable.
+- Add new `address template` to email alert.
```

Comment:
In the email [example TICKscript](https://docs.influxdata.com/kapacitor/v1.6/nodes/alert_node/#email), 
is it accurate to add the following as an example:

`.email('admin@example.com')`
` .toTemplates('oncall@example.com')`

And say that this passes the specified email addresses to the `email` property?

Reply:
yeah, I'm really unclear about this new capability and how really to take advantage of it.  We need a better description and example within the docs.

Reply:
Per Emrys - Previously, address had to be hardcoded - customer wanted to derive template for email address directly from data as opposed to script.